guides/installation/requirements.md

```diff
@@ -11,13 +11,16 @@ Before installing Shopware 6, take a quick look at the requirements below to che
 
 ## Operating System
 
-Shopware 6 is currently only supported on linux and macOS setups.
-Windows is only supported inside WSL 2.
+Shopware 6 is currently only supported on any Unix operating system. Windows is only supported inside WSL 2 or Docker.
 
 ## Versions
 
 You can use these commands to check your actual environment:
 
+::: info
+On many shared hosting environments, you have multiple PHP versions installed. Make sure that you use the correct PHP binary and often CLI and FPM have different `php.ini` files.
```

Comment:
maybe we could add a little bit more explanation on this error in the `troubleshooting guide` and link it here?